Sources/Core/HardwareBreakpointManager.cpp

```diff
@@ -102,6 +104,44 @@ ErrorCode HardwareBreakpointManager::disableLocation(Site const &site,
   return error;
 }
 
+// workaround for hardware readonly watchpoints not being implememted in
+// hardware ¯\_(ツ)_/¯ https://github.com/facebook/ds2/issues/257
```

Comment:
I don't think this really qualifies as a workaround, since the lack of hardware implementation is an intentional feature. You could just say that this is a software implementation of read-only watchpoints or something.